css/pages/navbar.css

```diff
@@ -130,6 +132,12 @@ nav li:hover a{
   
   }
 
-      
+.active{
+  background-color:#041187;
+  
+}  
+.active-link{
+  color:white;
+}   
```

Comment:
Instead of having to put 2 CSS selectors, 

Let's do something like this please:

```css
li.active a {
  color:white;
}

```

Reply:
.active and .active a ?
Like that na?